pkg/minikube/localpath/localpath.go

```diff
@@ -67,6 +67,11 @@ func EventLog(name string) string {
 	return filepath.Join(Profile(name), "events.json")
 }
 
+// AuditLog returns the path to the audit log.
```

Comment:
Can you update the docstrings of this and EventLog to clarify the difference between the two? It isn't obvious to a reader what gets saved in what.

Reply:
Updated comment to clarify difference between funcs